src/main/java/fm/last/moji/impl/FileUploadOutputStream.java

```diff
@@ -90,12 +92,17 @@ public void close() throws IOException {
           delegate.close();
         } finally {
           try {
-            String message = httpConnection.getResponseMessage();
             int code = httpConnection.getResponseCode();
             if (HttpURLConnection.HTTP_OK != code && HttpURLConnection.HTTP_CREATED != code) {
-              throw new IOException(code + " " + message);
-            } else {
-              log.debug("Status: HTTP {} - {}", code, message);
+              InputStream is;
+              if (200 <= httpConnection.getResponseCode() && httpConnection.getResponseCode() <= 299) {
+                is = httpConnection.getInputStream();
+              } else {
+                is = httpConnection.getErrorStream();
+              }
+              String body = IOUtils.toString(is);
```

Comment:
Yes, the response body might result in garbage, insufficient, redundant or useless. In my example, the request body is also not take effect for debugging since the information it provided has been shown by "code", "message" and "peer".

The intention here is to do "best effort" to get insight into error come from store/webdav server. Since I cannot assume/constraint which part of response will help for trouble shooting , the code here captures as much information as possible, includes connection peer, response code, message and body, and hope it can help user to debugging.

Since I do not have a good example here that the response body can help for debugging, maybe we can only add the "peer" information in this PR? What do you think?